test/binaries/test-perl-5.26.2.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+
+int main() {
+  printf("This program is designed to test the cve-bin-tool checker.");
+  printf("It outputs a few strings normally associated with perl 5.26.2.");
+  printf("They appear below this line.");
+  printf("------------------");
+  printf("An unknown option was passed in to perl");
+  printf("CLIENT perl 5.26.2");
```

Comment:
I think this was cut and pasted from libcurl and you didn't quite change the signature enough.  You're looking for a line that looks like  `Perl: 5.26.2` to match the regex you gave.